rocAL/rocAL_pybind/example/new_api/READERS_TEST_FILE.sh

```diff
@@ -0,0 +1,251 @@
+
+CURRENTDATE=`date +"%Y-%m-%d-%T"`
+
+# Mention the number of gpus
+gpus_per_node=4
+
+# Mention Batch Size
+batch_size=10
+
+# python version
+ver=$(python -V 2>&1 | sed 's/.* \([0-9]\).\([0-9]\).*/\1\.\2/')
+
+
+####################################################################################################################################
+# USER TO MAKE CHANGES HERE FOR TEST
+# Make the respective " Pipeline " to test equal to 1
+rocAL_api_python_unittest=1
+rocAL_api_coco_pipeline=1
+rocAL_api_caffe_reader=1
+rocAL_api_caffe2_reader=1
+rocAL_api_tf_classification_reader=1
+rocAL_api_tf_detection_pipeline=1
+rocAL_api_video_pipeline=1
+####################################################################################################################################
+
+
+
+
+####################################################################################################################################
+if [[ rocAL_api_python_unittest -eq 1 ]]; then
+
+    # Mention dataset_path
+    data_dir=$ROCAL_DATA_PATH/rocal_data/images_jpg/labels_folder/
+
+
+    # rocAL_api_python_unittest.py
+    # By default : cpu backend, NCHW format , fp32
+    # Please pass image_folder augmentation_nanme in addition to other common args
+    # Refer rocAL_api_python_unitest.py for all augmentation names
+    python$ver rocAL_api_python_unittest.py \
+        --image-dataset-path $data_dir \
+        --augmentation-name snow \
+        --batch-size $batch_size \
+        --display \
+        --NHWC \
+        --local-rank 0 \
+        --world-size $gpus_per_node \
+        --num-threads 1 \
+        --num-epochs 2 2>&1 | tee -a run.log.rocAL_api_python_unittest.${CURRENTDATE}
+fi
+####################################################################################################################################
+
+
+####################################################################################################################################
+if [[ rocAL_api_coco_pipeline -eq 1 ]]; then
+
+    # Mention dataset_path
+    data_dir=$ROCAL_DATA_PATH/coco/coco_10_img/val_10images_2017/
```

Comment:
@swetha097 The path is missing "/rocal_data/". Causes an error

Reply:
@LakshmiKumar23 , we need to export ROCAL_DATA_PATH=/Absolute/Path/Of/rocal_data/
[](https://github.com/swetha097/MIVisionX-data/tree/rocal_data_samples/rocal_data)